Objects/stringlib/asciilib.h

```diff
@@ -21,6 +21,7 @@
 #define STRINGLIB_CHECK          PyUnicode_Check
 #define STRINGLIB_CHECK_EXACT    PyUnicode_CheckExact
 #define STRINGLIB_MUTABLE 0
+#define STRINGLIB_FAST_MEMCHR    memchr
```

Comment:
I believe this should also go in stringlib/stringdefs.h and stringlib/UCS1lib.h as well, so that unicode objects with `128<=maxchar<255` and bytes objects still use memchr.

Reply:
In in V2.

Reply:
> Is that protocol for python patches or do you want a separate commits for all review fixes then
rebase at the end?

All PRs to CPython are squash-merged, so the precise way you manage the branch doesn't matter too much, but the GitHub UI doesn't cope very well with force-pushes -- I'd advise you to prefer merging instead.

A

Reply:
Looks like `bytes_methods.c` and `bytearrayobject.c` need these as well.
```
grep "define STRINGLIB_SIZEOF_CHAR"
```
```
Objects/bytearrayobject.c:#define STRINGLIB_SIZEOF_CHAR 1
Objects/bytes_methods.c:#define STRINGLIB_SIZEOF_CHAR 1
Objects/stringlib/asciilib.h:#define STRINGLIB_SIZEOF_CHAR    1
Objects/stringlib/stringdefs.h:#define STRINGLIB_SIZEOF_CHAR    1
Objects/stringlib/ucs1lib.h:#define STRINGLIB_SIZEOF_CHAR    1
Objects/stringlib/ucs2lib.h:#define STRINGLIB_SIZEOF_CHAR    2
Objects/stringlib/ucs4lib.h:#define STRINGLIB_SIZEOF_CHAR    4
```

Reply:
Fixed in V3.